### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by thaJeztah**
See individual commits for descriptions

### CODE REVIEW

**Comment by thaJeztah** [comment]
ping @justincormack @endophage @cyli PTAL

**Comment by justincormack** [comment]
ping @cyli @endophage

**Comment by justincormack** [comment]
Needs a rebase

**Comment by thaJeztah** [comment]
rebased; hopefully it's still green (otherwise I might have to re-vendor with the latest version of vndr if there's `go.mod` files in any of the dependency repositories)

**Comment by thaJeztah** [comment]
Still green 👍

File: vendor.conf [hunk 1]
```diff
@@ -1,57 +1,57 @@
...
-github.com/docker/go-connections    7395e3f8aa162843a74ed6d48e79627d9792ac55
+github.com/docker/go-connections    7395e3f8aa162843a74ed6d48e79627d9792ac55 # v0.4.0
 github.com/docker/go                d30aec9fd63c35133f8f79c3412ad91a3b08be06
 github.com/docker/go-metrics        399ea8c73916000c64c2c76e8da00ca82f8387ab
-github.com/dvsekhvalnov/jose2go     f21a8cedbbae609f623613ec8f81125c243212e6   # v1.3
-github.com/go-sql-driver/mysql      a0583e0143b1624142adab07e0e97fe106d99561   # v1.3
-github.com/gorilla/mux              53c1911da2b537f792e7cafcb446b05ffe33b996   # v1.6.1
+github.com/dvsekhvalnov/jose2go     f21a8cedbbae609f623613ec8f81125c243212e6 # v1.3
+github.com/go-sql-driver/mysql      a0583e0143b1624142adab07e0e97fe106d99561 # v1.3
+github.com/gorilla/mux              a7962380ca08b5a188038c69871b8d3fbdf31e89 # v1.7.0
```
Comment (thaJeztah) [inline vendor.conf]: This one may need an update https://github.com/gorilla/mux/compare/v1.7.0...v1.7.2

File: vendor.conf [hunk 2]
```diff
@@ -1,57 +1,57 @@
...
 github.com/jinzhu/gorm              5409931a1bb87e484d68d649af9367c207713ea2
 github.com/jinzhu/inflection        1c35d901db3da928c72a72d8458480cc9ade058f
 github.com/lib/pq                   0dad96c0b94f8dee039aa40467f767467392a0af
-github.com/mattn/go-sqlite3         6c771bb9887719704b210e87e934f08be014bdb1   # v1.6.0
-github.com/miekg/pkcs11             6120d95c0e9576ccf4a78ba40855809dca31a9ed
+github.com/mattn/go-sqlite3         6c771bb9887719704b210e87e934f08be014bdb1 # v1.6.0
+github.com/miekg/pkcs11             553cfdd26aaafe851ca66a5e8015f0decb6b5a1e
```
Comment (thaJeztah) [inline vendor.conf]: Looks like they're now tagging releases; I'll update to use v1.0.2; https://github.com/miekg/pkcs11/compare/553cfdd26aaafe851ca66a5e8015f0decb6b5a1e...v1.0.2

File: vendor.conf [hunk 3]
```diff
@@ -1,57 +1,57 @@
 github.com/Shopify/logrus-bugsnag   6dbc35f2c30d1e37549f9673dd07912452ab28a5
-github.com/sirupsen/logrus          f006c2ac4710855cf0f916dd6b77acf6b048dc6e  # v1.0.3
+github.com/sirupsen/logrus          8bdbc7bcc01dcbb8ec23dc8a28e332258d25251f # v1.4.1
 github.com/bugsnag/bugsnag-go       13fd6b8acda029830ef9904df6b63be0a83369d0
-github.com/bugsnag/panicwrap	    e2c28503fcd0675329da73bf48b33404db873782
-github.com/bugsnag/osext	    0dd3f918b21bec95ace9dc86c7e70266cfc5c702
+github.com/bugsnag/panicwrap        e2c28503fcd0675329da73bf48b33404db873782
```
Comment (thaJeztah) [inline vendor.conf]: These now do releases as well; this one is one commit ahead of v1.0.0; https://github.com/bugsnag/panicwrap/compare/1.0.0...e2c28503fcd0675329da73bf48b33404db873782 - one behind v1.1.0; https://github.com/bugsnag/panicwrap/compare/e2c28503fcd0675329da73bf48b33404db873782...1.1.0

v1.2.0 now un-forked  github.com/bugsnag/osext, I see https://github.com/bugsnag/panicwrap/compare/e2c28503fcd0675329da73bf48b33404db873782...v1.2.0

**Comment by thaJeztah** [comment]
@justincormack @HuKeping just rebased this again to bring it up to date with master

I can work on a follow-up to do the updates I mentioned

**Pull request merged**